lib/pages/dialer/pip/dismiss_keyboard.dart

```diff
@@ -1,5 +1,9 @@
 import 'package:flutter/material.dart';
 
 void dismissKeyboard(BuildContext context) {
-  FocusScope.of(context).requestFocus(FocusNode());
+  final FocusScopeNode currentFocus = FocusScope.of(context);
```

Comment:
```suggestion
  FocusManager.instance.primaryFocus?.unfocus();
```
No need context anymore

Reply:
thank u, I modified it